### ISSUE TITLE
Combobox paint (style) is hardcoded

### ISSUE THREAD

**Issue by azerupi** [MrKepzie/Natron#1003]
I am revamping the GUI through the stylesheet but I can't style the ComboBox. I have noticed you made a custom combobox based on a QFrame.

What is the reason behind that choice and not sticking whit the default QComboBox?

**Comment by MrKepzie** [comment]
The QComboBox has style issues on OSX with rounded corners of the drop-down menu and do not allow cascading menus.
Maybe the ComboBox code could be enhanced to take the style into account.

**Comment by devernay** [comment]
That's a great initiative!
I don't remember, and I don't see any comment in the code about it.
@MrKepzie certainly knows (cascading menus maybe?).
I agree that is in general a bad idea to make new widgets, but probably we has no other choice in this case.
We should at least make them style-able.

**Comment by azerupi** [comment]
@devernay Thanks :)
I just barely started, when I have something to show I will post it on the Forum for reviews.

I tried to fiddle a little with the `paintEvent` method but it has been a long time since have played with Qt. 

I found this resource about custom widgets and styling: https://wiki.qt.io/Qt_Style_Sheets_and_Custom_Painting_Example 
I will try to play with it a little and see if it is possible. Have any of you experience with this?
